src/main/java/tasklist/Task.java

```diff
@@ -0,0 +1,25 @@
+package tasklist;
+
+public abstract class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
```

Comment:
I believe isDone is already assigned default value of false by Java? So think can omit the setting of the variable again in the Task constructor?

Reply:
Thanks for the advice, will improvise the code accordingly!